homeassistant/components/modbus/modbus.py

```diff
@@ -198,10 +199,11 @@ async def async_setup(self):
         # Start counting down to allow modbus requests.
         if self._config_delay:
             self._async_cancel_listener = async_call_later(
-                self.hass, self._config_delay, self.async_end_delay
+                self.hass, self._config_delay, self.callback_end_delay
             )
 
-    async def async_end_delay(self, args):
+    @callback
+    def callback_end_delay(self, args):
```

Comment:
Our standard is to prefix all function names with `async_` if they should run in the event loop. It doesn't mean that they are coroutine functions.

```suggestion
    def async_end_delay(self, args):
```

Reply:
That is a bit of a strange standard....but standards are to be followed 😄 Done

Reply:
You can read more here:
https://developers.home-assistant.io/docs/asyncio_categorizing_functions

Reply:
thanks it is nice to know the reason, this is one I am likely to see in reviews.